lib/content-services/search/components/search-filter/search-filter.component.ts

```diff
@@ -161,12 +161,16 @@ export class SearchFilterComponent implements OnInit {
 
             this.responseFacetQueries = new ResponseFacetQueryList(facetQueries, this.facetQueriesPageSize);
 
-            const expandedFields = this.responseFacetFields.filter(field => field.expanded).map(field => field.label);
+            const expandedFields = this.responseFacetFields
+                .filter(field => field.expanded)
+                .map(field => field.label);
 
             this.responseFacetFields = (context.facetsFields || []).map(
                 field => {
+                    const settings = this.queryBuilder.getFacetField(field.label);
+
                     field.label = this.translationService.instant(field.label);
-                    field.pageSize = field.pageSize || 5;
+                    field.pageSize = field.pageSize || settings.pageSize || 5;
```

Comment:
Can we assign this value 5 to a constant with a name?

Reply:
updated to constant, thanks for pointing out